### DISCUSSION TITLE
Specie list preview for  BirdNET_GLOBAL_6K_V2.4_Model_FP16 (2023)

### DISCUSSION THREAD

**Discussion by gutocarv** [Nachtzuster/BirdNET-Pi#253] [Q&A]
Hi all
When I select the _BirdNET_GLOBAL_6K_V2.4_Model_FP16 (2023)_ model and enable the _"list preview"_ feature, does it display all bird species predicted by eBird for the selected region/coordinates, or only those species that both eBird predicts and the BirdNET model is capable of recognizing?

**Comment by alexbelgium** [comment]
The list preview shows, for the threshold that you have defined, the species that the model will return.
It is the same content as the `PREDICTED_SPECIES_LIST ` variable from the server.py. If the model returns a value not from this list, it won't be shown.

https://github.com/Nachtzuster/BirdNET-Pi/blob/e1c25b816595dd95cf2750cbec232dd5617f81b9/scripts/server.py#L338-L339

It allows the set the threshold depending on what you want : for night prey birds for example you need to set the threshold extremely low (and accept false positives - which then puts the emphasis on excellent mic/amp/gain settings to mitigate this), but for common day birds a higher threshold value is fine
